### PR TITLE
Fix for SendAction execution in simulator

### PR DESCRIPTION
**Pull request by rowanseymour**
For messages sent from SendAction, `@contact` refers to the contact the message is being sent to rather than the contact in the flow. Since we might be sending to a group of contacts, we shouldn't resolve `@contact` for simulator logging.

Should fix https://app.getsentry.com/nyaruka/textit/group/90106531/

### CODE REVIEW

File: temba/flows/models.py [hunk 1]
```diff
@@ -1221,7 +1223,7 @@ def value_wrapper(value):
 
         context = dict(flow=flow_context, channel=channel_context, step=message_context, extra=run_context)
         if contact:
-            context['contact'] = contact
```
Comment (rowanseymour) [inline temba/flows/models.py]: This was putting an actual contact object in the context

**Comment by norkans7** [comment]
Looks good

**Comment by rowanseymour** [comment]
Also includes update to expressions 1.0.11 which has this fix https://github.com/rapidpro/expressions/commit/a96a6d8fd8c845f8c6eaaa222083b5b26631b278 for related error https://app.getsentry.com/nyaruka/rapidpro/group/90159483/

**Comment by nicpottier** [comment]
Noice.

**Pull request merged**